go.mod

```diff
@@ -2,6 +2,6 @@ module github.com/fsnotify/fsnotify
 
 go 1.16
 
-require golang.org/x/sys v0.0.0-20210630005230-0f9fa26af87c
+require golang.org/x/sys v0.0.0-20210817134402-fefb4affbef3
```

Comment:
This is a minimum version to pick up the event ports code. Should we update to the latest? Some other specific version e.g. one used in a particular release of golang?

Reply:
probably latest? if everything still passes in our test matrix